src/cbl-mariner/2.0/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
+
+# Install Helix Dependencies
+
+ENV LANG=en_US.utf8
+
+RUN tdnf install --setopt tsflags=nodocs --refresh -y \
+         ca-certificates-microsoft \
+         gcc \
+         icu \
+         iputils \
+         python3-pip \
+         shadow-utils \
+         tzdata \
+         which \
+    && tdnf clean all
+
+# Install MsQuic - hack until packages are pushes to mariner repository
+# When that is done, this should go to section above.
+# Mariner already has MS repos and signing keys.
+RUN rpm -i https://packages.microsoft.com/centos/7/prod/libmsquic-2.1.4-1.x86_64.rpm
+
+RUN ln -sf /usr/bin/python3 /usr/bin/python && \
+    python -m pip install --upgrade pip==21.1.2 && \
+    python -m pip install virtualenv==16.6.0 && \
+    python -m pip install --upgrade setuptools && \
+    pip download --no-deps helix-scripts --index-url https://dnceng.pkgs.visualstudio.com/public/_packaging/helix-client-prod/pypi/simple && \ 
+    pip install ./helix_scripts-*-py3-none-any.whl && \
+    rm ./helix_scripts-*-py3-none-any.whl
+
+# create helixbot user and give rights to sudo without password
+RUN /usr/sbin/useradd -c '' --uid 1000 --shell /bin/bash --groups adm helixbot && \
+    chmod 755 /root && \
+    echo "helixbot ALL=(ALL)       NOPASSWD: ALL" > /etc/sudoers && \
```

Comment:
As @ChadNedzlek  pointed out recently this should probably be >> to not stomp the contents of sudoers

Reply:
make sense. updated.